test/image/mocks/bar_display_height_zero_only_line_width.json

```diff
@@ -0,0 +1,340 @@
+                0,
+                null,
+                null,
+                1
+            ],
+            "text": [
+                0,
+                0,
+                null,
+                null,
+                1
+            ],
+            "textposition": "auto",
+            "insidetextanchor": "middle",
+            "cliponaxis": false,
+            "xaxis": "x2",
+            "yaxis": "y2"
+        },
+        {
+            "type": "bar",
+            "marker": {
+                "line": {
+                    "width": 0
+                }
+            },
+            "orientation": "h",
+            "y": [
+                "A",
+                "b",
+                "c",
+                "d",
+                "E"
+            ],
+            "x": [
+                0,
+                0,
+                null,
+                null,
+                1
+            ],
+            "text": [
+                0,
+                0,
+                null,
+                null,
+                1
+            ],
+            "textposition": "auto",
+            "insidetextanchor": "middle",
+            "cliponaxis": false,
+            "xaxis": "x3",
+            "yaxis": "y3"
+        },
+        {
+            "type": "bar",
+            "marker": {
+                "line": {
+                    "width": [10, 0, 10, 0, 10]
+                }
+            },
+            "orientation": "h",
+            "y": [
+                "A",
+                "b",
+                "c",
+                "d",
+                "E"
+            ],
+            "x": [
+                0,
+                0,
+                null,
+                null,
+                1
+            ],
+            "text": [
+                0,
+                0,
+                null,
+                null,
+                1
+            ],
+            "textposition": "auto",
+            "insidetextanchor": "middle",
+            "cliponaxis": false,
+            "xaxis": "x3",
+            "yaxis": "y3"
+        },
+        {
+            "type": "bar",
+            "marker": {
+                "line": {
+                    "width": 0
+                }
+            },
+            "orientation": "h",
+            "y": [
+                "A",
+                "b",
+                "c",
+                "d",
+                "E"
+            ],
+            "x": [
+                0,
+                0,
+                null,
+                null,
+                1
+            ],
+            "text": [
+                0,
+                0,
+                null,
+                null,
+                1
+            ],
+            "textposition": "auto",
+            "insidetextanchor": "middle",
+            "cliponaxis": false,
+            "xaxis": "x4",
+            "yaxis": "y4"
+        },
+        {
+            "type": "bar",
+            "marker": {
+                "line": {
+                    "width": [10, 0, 10, 0, 10]
```

Comment:
Looks like this mock only has array `marker.line.width` value. Could you add one trace that test scaler `marker.line.width`? 

Maybe you can leave this mock alone and simply add another mock using https://codepen.io/MojtabaSamimi/pen/bXbmba , so that we also test the relative-stacked case.